androidsdk/src/main/java/com/wootric/androidsdk/network/tasks/WootricRemoteRequestTask.java

```diff
@@ -111,7 +111,8 @@ protected void onPostExecute(String response) {
             decodedString = new String(response.getBytes("ISO-8859-1"), "UTF-8");
             onSuccess(decodedString);
         } catch (UnsupportedEncodingException e) {
-            e.printStackTrace();
+            onSuccess(response);
+        } catch (NullPointerException e) {
```

Comment:
Yes, that's the process we have so let's stick to it.

On Sat, Jun 25, 2016 at 1:52 PM Diego <notifications@github.com> wrote:

> In
> androidsdk/src/main/java/com/wootric/androidsdk/network/tasks/WootricRemoteRequestTask.java
> <https://github.com/Wootric/WootricSDK-Android/pull/28#discussion_r68493710>
> :
>
> > @@ -111,7 +111,8 @@ protected void onPostExecute(String response) {
> >              decodedString = new String(response.getBytes("ISO-8859-1"), "UTF-8");
> >              onSuccess(decodedString);
> >          } catch (UnsupportedEncodingException e) {
> > -            e.printStackTrace();
> > +            onSuccess(response);
> > +        } catch (NullPointerException e) {
>
> Should I merge? 👍
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/Wootric/WootricSDK-Android/pull/28/files/0f41288741bfa5be753b7dee0d99507d968cde93#r68493710>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAVkDKOs_4MqkXTbpldsOYCTkeIUOcoQks5qPZUKgaJpZM4I-api>
> .
>